### ISSUE TITLE
Updating cacheError

### ISSUE THREAD

**Issue by legopin** [tldr-pages/tldr-node-client#236]
## Expected behaviour
```
npm install -g tldr
tldr tar
```
First time loading the page, should download the page data and update local cache.

## Actual behaviour
Throws error Updating chaceError:Cannot update from http://tldr-pages.github.io/assets/tldr.zip

## Log, debug output
```
tldr ls
⠏ Page not found. Updating cacheError: Cannot update from http://tldr-pages.github.io/assets/tldr.zip
    at Extract.extractor.on (/Users/meepshop/.nvm/versions/node/v8.9.4/lib/node_modules/tldr/lib/remote.js:33:14)
    at emitOne (events.js:116:13)
    at Extract.emit (events.js:211:7)
    at Parse.<anonymous> (/Users/meepshop/.nvm/versions/node/v8.9.4/lib/node_modules/tldr/node_modules/node-unzip-2/lib/extract.js:28:10)
    at emitOne (events.js:116:13)
    at Parse.emit (events.js:211:7)
    at /Users/meepshop/.nvm/versions/node/v8.9.4/lib/node_modules/tldr/node_modules/node-unzip-2/lib/parse.js:64:12
    at runCallback (timers.js:789:20)
    at tryOnImmediate (timers.js:751:5)
    at processImmediate [as _immediateCallback] (timers.js:722:5)
```

## Environment
- Operating system - OSX 10.12.6
- NodeJS version: v8.9.4

**Comment by agnivade** [comment]
2 is also done.

And DNS changes have also propagated. Issue should be fixed.

You can also upgrade to 3.2.5 if DNS hasn't still expired for you.

**Comment by legopin** [comment]
Awesome it's fixed now. 
Thanks for the quick response
